File: W3/Jewels and Stones.dart
class Solution {
  int numJewelsInStones(String jewels, String stones) {
    int c=0;    //counter
    for(int i=0;i<jewels.length;i++){
      for(int j=0;j<stones.length;j++){
        if(stones[j]==jewels[i])
          c++;
      }
    }
    return c;
  }
}

File: W3/Jewels and Stones.cpp
class Solution {
public:
    int numJewelsInStones(string jewels, string stones) {
        int c=0;
        map<char,int> fr;
        for(char i:stones)
          fr[i]++;
        for(char i:jewels)
          c += fr[i];
        
        return c;
    }
};